.github/SECURITY.md

```diff
@@ -0,0 +1,13 @@
+# Security Policy
+
+## Supported Versions
+
+All versions of our software are supported by this policy. However only the most recent version is supported for vulnerability reporting on its repository and is eligible for fixes and new releases. If a vulnerability is found in older versions of the software, a vulnerability report should instead be made on *dependant* software, so that an update may be issued.
+
+## Reporting a Vulnerability
+
+If a security vulnerability is discovered, we ask for a non-public disclosure. ***The repository "Issues" tracker is fully public.*** Security vulnerabilities can be privately disclosed on the repository's "Security" page (https://github.com/PretendoNetwork/NAME/security/advisories/new). Maintainers will be automatically notified and any non-notified, relevant, contributors will be made aware internally.
```

Comment:
Relative link?

Reply:
Does this get rendered on each repository as well?

Reply:
I'm not 100% sure how it displays in the repo's code with a global .github. I know it displays on the security page though - check my repo at https://github.com/MatthewL246/hello-world/security.

Reply:
Ah, that's unfortunate. It would be great if GitHub provided a generic way to insert the repository name in the template, but that's too bad.